meteor/client/ui/RundownView/RundownTiming/BreakWallTime.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from 'classnames'
+import React from 'react'
+import { SegmentUi } from '../../SegmentTimeline/SegmentTimelineContainer'
+import { RundownUtils } from '../../../lib/rundown'
+import { useTranslation } from 'react-i18next'
+
+interface IBreakWallTimeProps {
+	segment: SegmentUi
+	className?: string
+	labelClassName?: string
+}
+
+/**
+ * A presentational component that will render a counter that will show the wall time that a break starts at.
+ * (Or ends at, if there's no start time but an end time is provided.)
+ * @function BreakWallTime
+ * @extends React.Component<IBreakWallTimeProps>
+ */
+export const BreakWallTime = function BreakWallTime(props: IBreakWallTimeProps): JSX.Element | null {
```

Comment:
I think this component should be renamed to reflect that it's not related to Breaks.